docs/arf.md

```diff
@@ -1711,13 +1711,23 @@ purposes:
     Wallet Instance. The WTE is described in \[Topic 9\]. The WTE has
     two main purposes:
 
+    -   It describes the capabilities and properties of the Wallet
+        Instance, the User device and the WSCD(s) and lets the Wallet Provider rerecognize its Wallet Instance.
+
+    -   Moreover, the WTE contains a WTE public key. This key can be used
+        to authenticate the Wallet Instance towards the Wallet Provider.
+
+3.  The EUDI Wallet Provider issues a Issuer Trust Evidence (ITE) to the
+    Wallet Instance and behaves similarly to the WTE. The ITE has
+    two main purposes:
+
     -   It describes the capabilities and properties of the Wallet
         Instance, the User device and the WSCD(s). This allows a PID
         Provider or an Attestation Provider to verify that the Wallet
         Instance complies with the Provider's requirements and therefore
         is fit to receive a PID or an attestation from the Provider.
 
-    -   Moreover, the WTE contains a WTE public key. During the issuance
+    -   Moreover, the ITE contains an ITE public key. During the issuance
```

Comment:
the text still seems to entirely depend on PoA for the actual keys used in PIDs/(Q)EAAs, I would suggest to also allow to directly associate the key used in a PID/(Q)EAA with a certain WSCD through an ITE.